config/sample.yml

```diff
@@ -42,6 +42,16 @@ messageStore: # Postgresql database configuration for message store
   password:
   url:
 
+contactDiscovery: # Configuration for interfacing with Contact Discovery Service cluster
+  accessKey:      # AWS SQS accessKey
+  accessSecret:   # AWS SQS accessSecret
+  queueUrl:       # AWS SQS queue url
+  serverApiUrl:   # CDS server-api base url
+  serverAptToken: # CDS server-api auth token
+  serverApiCaCertificate: # CDS server-api TLS certificate trust root
```

Comment:
The above three seem like they could be a part of a directory object

Reply:
done in 8811e29bd3e00e8c272d7650b9abb3622eb13089